content/en/docs/zero-code/java/spring-boot-starter/out-of-the-box-instrumentation.md

```diff
@@ -20,6 +20,12 @@ Out of the box instrumentation is available for several frameworks:
 
 ## Common instrumentation configuration
 
+To enable only specific instrumentations, you can disable all the
```

Comment:
- should be part of previous section 
- section should have a title, e.g. "Suppressing specific instrumentation"
- section needs an explanation like here: https://opentelemetry.io/docs/zero-code/java/agent/configuration/#suppressing-specific-agent-instrumentation

Reply:
Change done as dicussed during our chat